.github/CODEOWNERS

```
@@ -13,10 +13,42 @@
 # This file was adapted from SciPy.
 
 # Each line is a file pattern followed by one or more owners.
+# Syntax follows .gitignore, see https://git-scm.com/docs/gitignore#_pattern_format
+# Headings ideally should correspond to domains of knowledge/expertise.
+
+# Beamforming
+/mne/beamformer @britta-wstnr
+*dics*.py @britta-wstnr  # related tutorials and examples
+*lcmv*.py @britta-wstnr
 
 # IO
 /mne/io/brainvision @sappelhoff
 /mne/export @sappelhoff
 
-# Beamforming
-/mne/beamformer/ @britta-wstnr
+# Viz
+/mne/viz @drammock
+/mne/viz/_brain @larsoner
+/tutorials/visualization @larsoner
+/examples/visualization @larsoner
+
+# Core sensor-space classes
+/mne/epochs.py @drammock
+/mne/evoked.py @drammock
+/mne/io/*.* @drammock
```

I don't think this will include things like `mne/io/fiff/raw.py`. If you want all of `mne/io` I think you just want
```suggestion
/mne/io @drammock
```
But maybe you did just want stuff like `mne/io/base.py`?

---

... ahh I see I need to read better, core sensor-space classes :+1: